src/ui-lib/sass/05-components/01-atoms/10-navigation/_00-block-link.scss

```diff
@@ -10,6 +10,7 @@
     &:visited {
       background-color: $grav-co-interactive-secondary;
       border-color: $grav-co-interactive-secondary;
+      border-radius: $grav-st-radius;
```

Comment:
Not sure if the rounding makes sense on the block links. They get used in the nav bar and since the bottom border is used to create a pseudo underline effect, it looks a little odd when the edges curve up.